### PR TITLE
magnus update to RTCP APP

### PR DESCRIPTION
**Pull request by fluffy**

### CODE REVIEW

**Comment by aboba** [comment]
LGTM.

**Comment by fluffy** [comment]
I'm in favour of merging this

**Comment by fluffy** [comment]
I fixed the typo Bernard pointed out - thanks

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -5884,8 +5884,11 @@ a=msid:-
           SSRC table, deliver the packet to the "m=" line associated
           with that SSRC.</t>
 
-          <t>If the packet is of type APP, deliver a copy of the packet
-          to each "m=" line.</t>
+           <t> The RTCP APP packet only specifies the source of the
+           packet. Thus this information can be related to any of the
+           "m=" lines, thus deliver a copy of the packet to each "m="
+           line. </t>
```
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: "As this information..., deliver"

Comment (fluffy) [inline draft-ietf-rtcweb-jsep.xml]: fixed

**Comment by aboba** [comment]
LGTM as well.

**Comment by juberti** [comment]
Will polish and land.

**Pull request merged**